tests/conftest.py

```diff
@@ -16,3 +25,64 @@ def setup_env(pytestconfig, request):
 
     if not running_unit_tests and environment in ['local', 'qa']:
         load_env_file(environment, file_string=f'config/{environment}.yaml')
+
+@pytest.fixture(scope='module')
```

Comment:
question on the module scope. does this run during the unit tests? 

we just want to run it during functional and integration tests